internal/command/meta.go

```diff
@@ -736,6 +736,27 @@ func (m *Meta) showDiagnostics(vals ...interface{}) {
 	}
 }
 
+const (
+	// StatePersistIntervalEnvVar is the environment variable that can be set
+	// to control the interval at which Terraform persists state. The interval
+	// itself defaults to 20 seconds.
+	StatePersistIntervalEnvVar = "TF_STATE_PERSIST_INTERVAL"
+)
+
+// StatePersistInterval returns the configured interval that Terraform should
+// persist statefiles to the desired backend. Backends may choose to override
+// the default value.
+func (m *Meta) StatePersistInterval() int {
+	if val, ok := os.LookupEnv(StatePersistIntervalEnvVar); ok {
+		interval, err := strconv.Atoi(val)
+		if err == nil {
+			return interval
```

Comment:
Should we add guards here - like, don't allow to specify negative values, and maybe put 20 as a minimum value? We really need only much higher values, like 600, 1200, etc.